twinkle.js

```diff
@@ -438,6 +438,7 @@ Twinkle.load = function () {
 		Twinkle.batchdelete();
 		Twinkle.batchprotect();
 		Twinkle.batchundelete();
+		Twinkle.block();
```

Comment:
Block is unusual in that it is a module admins are likely to want to use frequently. The existing 6 admin-only modules are rather specialised or little-used (I don't think anyone ever uses batch undelete), hence why they are at the bottom. I would suggest to group block with the user-related modules (inside the necessary if statement).